app/2.0/docs/devguide/events.md

```diff
@@ -56,9 +56,21 @@ ready() {
 }
 ```
 
-The previous example uses an arrow function to ensure the listener is called with the element as the
-`this` value. You can also use `bind` to create a bound instance of the listener function. This can
-be  useful if you need to remove the listener.
+```js
+ready() {
+  super.ready();
+  this.addEventListener('click', this._onClick);
+}
+
+_onClick(event) { /* ... */ }
+```
+
+The `this` value in `_onClick()` in the example above is the custom element because the callback is
+called with the event's current target as `this`. Also, the use of `ready()` is important, because,
```

Comment:
The first sentence is kind of half the story—or, to put it another way, it assumes that you _do_ know how currentTarget is defined (i.e., that it's always the node that you added the listener to), but that you _don't_ know that currentTarget and this are the same. 

So we could say here something like:

By default, an event handler is called with the `this` value set to the event's _current target_. The current target is always equal to the element that the event listener is attached to, in this case, the custom element itself.

Reply:
Good point. I'll rephrase that, so that it's clearer.